### PR TITLE
Added missing flag, and remove useless flag

### PR DESCRIPTION
**Pull request by Patrickooos**

### CODE REVIEW

**Comment by BPatrickooos** [comment]
No big changes, just added missing user flags

File: DSharpPlus/Enums/User/UserFlags.cs [hunk 1]
```diff
@@ -81,11 +81,6 @@ public enum UserFlags
         /// </summary>
         TeamUser = 1 << 10,
 
-        /// <summary>
-        /// Whether the user is an official system user.
-        /// </summary>
-        System = 1 << 12,
```
Comment (OoLunar) [inline DSharpPlus/Enums/User/UserFlags.cs:87]: Why was this removed?
